snmp/tests/test_profiles.py

```diff
@@ -65,16 +65,28 @@ def test_load_profiles(caplog):
         caplog.clear()
 
 
-def run_profile_check(recording_name):
+def run_profile_check(recording_name, profile=None):
     """
     Run a single check with the provided `recording_name` used as
     `community_string` by the docker SNMP endpoint.
     """
+
     instance = common.generate_instance_config([])
 
     instance['community_string'] = recording_name
     instance['enforce_mib_constraints'] = False
     check = SnmpCheck('snmp', {}, [instance])
+
+    # First, see if recording name is a profile, then use profile as definition.
+    prof_def = check.profiles.get(recording_name)
+    if profile:
+        prof_def = check.profiles.get(profile)
```

Comment:
```suggestion
    if profile:
        prof_def = check.profiles.get(profile)
    else:
        prof_def = check.profiles.get(recording_name)
```

It's a nit - but this might be a little more explicit.

Reply:
Makes sense!